packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -25,7 +25,10 @@ const Curve _modalBottomSheetCurve = decelerateEasing;
 const double _minFlingVelocity = 700.0;
 const double _closeProgressThreshold = 0.5;
 
+/// A callback for when the user begins dragging the bottom sheet.
```

Comment:
Ditto

Reply:
Not quite sure what context to provide on this one...  @HansMuller: can you advise?

Reply:
Used by [BottomSheet.onDragStart].

Reply:
Thanks!